Add information about pages in incidents (#30837)
DataDog/documentation: content/en/service_management/incident_management/incident_settings/notification_rules.md

1. Navigate to [Incident Settings Notification Rules][1].
1. Click **New Rule**.
1. Configure the conditions for this notification rule. Select the incident property field `key:value` pairs you want notifications to be sent for. By default, these filters are empty, and a notification rule triggers when an incident is declared or when it's attributes are update.

> 3. **Notify**: Select your notification recipients.

Reviewer: ```suggestion
1. **Notify**: Select your notification recipients.
```

    - Notifications can be sent to any of Datadog's existing [notification integrations][2].

>     - Notification can be sent to existing [Datadog On-Call teams][4] using the `@oncall` handle.

Author: ```suggestion
    - Notifications can be sent to existing [Datadog On-Call teams][4] using the `@oncall` handle.
```

>     - Notifications can be sent to existing [Datadog On-Call teams][4] using the `@oncall` handle.

Reviewer: ```suggestion
    - Notifications can be sent to existing [Datadog On-Call teams][4] using `@oncall-` handles.
```

>     - If you want to notify a recipient's mobile device, select the option for their name that includes **(Mobile Push Notification)**. The recipient must have enabled notifications in the [Datadog mobile app][3] for this option to appear.

Reviewer: While we're in here, an addition:

```suggestion
    - If you want to notify a recipient's mobile device, select the option for their name that includes **(Mobile Push Notification)**. The recipient must have enabled notifications in the [Datadog mobile app][3] for this option to appear.
    - If you want to notify the incident Slack channel, use handle `@incident-slack-channel`.
```
